cmake/dependencies/PICSAR.cmake

```diff
@@ -69,6 +69,10 @@ function(find_picsar)
         mark_as_advanced(PXRMP_QED_TABLEGEN)
         mark_as_advanced(PXRMP_QED_OMP)
         mark_as_advanced(PXRMP_QED_TEST)
+        mark_as_advanced(PXRMP_BOOST_TEST_DYN_LINK)
+        mark_as_advanced(PXRMP_DPCPP_FIX)
+        mark_as_advanced(PXRMP_QED_TOOLS)
```

Comment:
Is `PXRMP_QED_TOOLS` generally not relevant to WarpX users?

For example, plotfile tools im AMReX we leave as regular option in case the users wants to build them, too.